### PR TITLE
docs: update color-picker JSDoc link

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** N/A

## Summary

Tidy up for color-picker's `value` doc.

This also works as a side-effect fix for a build error in Stencil v2.18. 🎉 cc @driskull

### CODE REVIEW

**Review by driskull — APPROVED** [review]
👍

File: src/components/color-picker/color-picker.tsx [hunk 1]
```diff
@@ -264,12 +264,13 @@ export class ColorPicker implements InteractiveComponent {
   /**
    * The color value.
```
Comment (geospatialem) [inline src/components/color-picker/color-picker.tsx:265]: What would ya think about the following below, in line with #4442 for the `value` prop? 

> The ~color~ component's value.

**Review by geospatialem — COMMENTED** [review]
Very nice! 🎨 Small consideration with the initial doc line, otherwise we can get this one in later, too.

File: src/components/color-picker/color-picker.tsx (hunk 1 above)
Comment (driskull) [reply src/components/color-picker/color-picker.tsx]: I changed it.

**Pull request merged**